### PR TITLE
PRO-2196 - Added ARKA wrapper as sub package

### PR DESCRIPTION
**Pull request by vignesha22**
## Description

- Added ARKA apis as a sub package module

## Types of changes
What types of changes does your code introduce?

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Other (please describe): 

## Further comments (optional)

-

### CODE REVIEW

File: src/sdk/paymaster/provider/arka.ts [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+  readonly queryParams: string;
+
+  constructor(chainId: number, apiKey: string, paymasterUrl: string) {
+    this.url = paymasterUrl;
+    this.queryParams = `?apiKey=${apiKey}&chainId=${chainId}`;
+    this.apiKey = apiKey;
+    this.chainId = chainId;
+  }
+
+  async getTokenPaymasterAddress(tokenSym: string) {
```
Comment (ch4r10t33r) [inline src/sdk/paymaster/provider/arka.ts:18]: Pls add comments prior to each method describing what they do.

File: src/sdk/paymaster/provider/arka.ts [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+    let response = null;
+    const entryPointAddress = '0x5FF137D4b0FDCD49DcA30c7CF57E578a026d2789';
```
Comment (ch4r10t33r) [inline src/sdk/paymaster/provider/arka.ts:20]: We need to make Arka compatible for both EP0.6.0 and EP0.7.0, so rename this variable as entryPointAddressV6.

**Review by ch4r10t33r — CHANGES_REQUESTED** [review]

**Review by ch4r10t33r — APPROVED** [review]
lgtm

**Review by kaushalrajbacancy — APPROVED** [review]
LGTM

**Pull request merged**